### ISSUE TITLE
VRM Export: Multiple materials with no texture + transparent texture exports invisible asset

### ISSUE THREAD

**Issue by madjin** [M3-org/CharacterStudio#147]
1242: seems that the horns become transparent in the exported vrm

[invisible-horns-1242.zip](https://github.com/user-attachments/files/15685707/invisible-horns-1242.zip)

![image](https://github.com/M3-org/CharacterStudio/assets/32600939/bc11001c-1eb5-42e9-ab02-927a9a469fa8)
![image](https://github.com/M3-org/CharacterStudio/assets/32600939/a5653555-401c-41f1-ac6e-6da7637b82c3)

![image](https://github.com/M3-org/CharacterStudio/assets/32600939/d25bd119-2ed5-4538-8b94-9509c99c77ba)

The horns do not have a texture, it's just base color white.

1535: horns in visible in vrm

[invisible-horns-1535.zip](https://github.com/user-attachments/files/15685713/invisible-horns-1535.zip)

![image](https://github.com/M3-org/CharacterStudio/assets/32600939/fc353c6b-b9d3-421a-961c-053834b5df7d)

**Comment by Benjythebee** [comment]
I tried debugging this but I can't help because I can't even reproduce a VRM with this issue.

I downloaded the horns, stripped away the skeleton and exported it clean as GLB. Then I applied it to a VRM0 skeleton and exported a new VRM that would work.

For VRM0, the VRM is straight up broken.
![image](https://github.com/M3-org/CharacterStudio/assets/38708022/1dddcdc1-e501-4825-a772-fdbac2e37c9c)

This is pretty common though. Any VRM0 that's exported with at least one mesh that has no texture will be completely broken, that's a bug and I'm not sure if there is a ticket for it on Github.

Now, I went ahead and tried to do the same on the drophunter since the drophunter avatar is a VRM1.
I export a new Drophunter avatar with the horns and open it on Blender. This time I'm seeing the avatar in blender, but it comes alongside to this error:

![image](https://github.com/M3-org/CharacterStudio/assets/38708022/eef58ef3-351e-4dee-bffb-39de2acba168)

so I'm not sure if it's because I have a bad `Horn.vrm` or if it's something else that's different.

FYI, I can open your 1535 avatar vrm with no errors
